### PR TITLE
Added Twitter Context

### PR DESCRIPTION
**Pull request by anishsarangi**
@vbanos I have made first commit on Twitter Context Screen. Changes are added in this PR.

### CODE REVIEW

**Comment by vbanos** [comment]
@anishsarangi I've found a serious bug: When I try `http://www.vbanos.gr`, the actual URL that is sent to the back-end is `vbanos.`. When I try `https://cnn.com/`, I get `cnn.co`.

1. Why do you truncate the initial `http(s)://`? Please do not process it, let the back-end do that.
2. Why do you try to truncate the last chars? Please avoid that.

**Comment by vbanos** [comment]
The appearance of the tweets is not very good.
Please check this out and try to use it:
https://developer.twitter.com/en/docs/twitter-for-websites/embedded-tweets/guides/css-for-embedded-tweets

Also, why the *whole* tweet text is a direct link? It shouldn't be.
Please see how embedded tweets appear in a website using the twitter widget, e.g. http://iskme.org/ and try to make it look the same in our case.
Thank you!

File: twitter.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                        <div class='container'>
+                            <img class='resize_fit_center'
+                            src='images/logo.gif' />
+                        </div>
+                    <div id="horizontal_line" class="extra_margin"></div>
+                </td>
+            </tr>
+        </table>
+        <div style="text-align: center">
+            <h2><strong>Tweets containing of Current URL</strong><br></h2>
```
Comment (vbanos) [inline twitter.html]: Tweets containg the current URL

Comment (anishsarangi) [reply twitter.html]: Thank you!!

**Comment by anishsarangi** [comment]
I am closing this PR and requesting a new one
Thank you !

**Pull request closed without merging**